src/components/Dashboard/FilingHistoryList.vue

```diff
@@ -627,16 +636,24 @@ export default {
     },
 
     /**
-     * Function to return a boolean if this specific filing is future affective.
-     *
+     * Whether this specific filing is a future affective COA.
      * @param filingType The type of the filing in the history list.
      * @param status The status of the filing in the history list.
-     * @returns A boolean indicating if the filing is future effective.
      */
     isCoaFutureEffective (filingType: string, status: string): boolean {
-      return (this.isBComp() &&
+      return (
+        this.isBComp() &&
         filingType === FilingTypes.CHANGE_OF_ADDRESS &&
-        status === FilingStatus.PAID)
+        this.isPaid(status)
```

Comment:
I guess this answers my previous question 😆

Reply:
Future effective COA are separate from FE IA filings, I'm pretty sure.

Reply:
I just reused the new helper function here for cleanliness. I think we might have more such helper functions and eventually put them in a mixin or something.